spec/command/trunk/push_spec.rb

```diff
@@ -170,6 +170,13 @@ def found_podspec_among_files(files)
         e = should.raise(Informative) { cmd.send(:validate_podspec) }
         e.message.should.include 'The spec did not pass validation, due to failure_reason.'
       end
+
+      it 'passes skip import validation' do
+        Validator.any_instance.expects(:skip_import_validation)
```

Comment:
@segiddins what am I doing so wrong and this fails?

```
Bacon::Error: not all expectations were satisfied
unsatisfied expectations:
- expected exactly once, not yet invoked: #<AnyInstance:Pod::Validator>.skip_import_validation(any_parameters)
```

I understand the error msg and printing out Validator shows that `skip_import_validation` is correctly set.......